101-bundle-json.md

```diff
@@ -196,6 +197,20 @@ The following OPTIONAL fields MAY be attached to an invocation image:
   - `os`: The operating system of the image
 - `mediaType`: The media type of the image
 
+### Resolving Image References
+
+With invocation images and images that use the OCI image format, the referenced image is of the format `REGISTRY/IMAGE:TAG@DIGEST`.
+
+`REGISTRY` is optional. `IMAGE` is required. `TAG` is optional. `DIGEST` is optional, but strongly recommended.
+
+- If `IMAGE` is not specified, the runtime MUST fail with an error.
+- When `DIGEST` is specified, the runtime MUST use the version specified by that digest and MUST fail if an image of that digest cannot be found.
+- When no `DIGEST` is specified, but a `TAG` is specified, the runtime MUST use an image with that tag or fail.
+- When neither `DIGEST` nor `TAG` is defined, the behavior is undefined.
+- If `REGISTRY` is missing, a runtime SHOULD assume `dockerhub.io` is the registry. This matches the behavior of existing clients.
+
+For images that are not OCI images, the digest MAY be embedded using the `@digest` notation above. However, for such formats, `TAG`, `NAME`, and `REGISTRY` remain undefined in this spec.
+
```

Comment:
Maybe add a statement that before signing/sharing trough a registry, all image references should be digested ?